server/channels/app/helper_test.go

```diff
@@ -260,7 +263,10 @@ func (th *TestHelper) InitBasic() *TestHelper {
 	// create users once and cache them because password hashing is slow
 	initBasicOnce.Do(func() {
 		th.SystemAdminUser = th.CreateUser()
-		th.App.UpdateUserRoles(th.Context, th.SystemAdminUser.Id, model.SystemUserRoleId+" "+model.SystemAdminRoleId, false)
+		_, appErr := th.App.UpdateUserRoles(th.Context, th.SystemAdminUser.Id, model.SystemUserRoleId+" "+model.SystemAdminRoleId, false)
+		if appErr != nil {
+			panic(appErr)
+		}
```

Comment:
Let's add a `testing.TB` field to `TestHelper` and use `require.Nil` here and in the rest of the file